msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/brokerapi/TestCase1561652.java

```diff
@@ -104,14 +104,7 @@ public void decodeSSOTokenAndVerifyNonce(@NonNull final String ssoToken,
                     .append('\n');
         }
         final String decodedToken = sb.toString();
-        if (decodedToken.contains("request_nonce")) {
-            final String[] str = decodedToken.split("request_nonce => ");
-            if (str.length > 1) {
-                Assert.assertEquals(str[1].trim(), nonce);
-            } else {
-                Assert.fail("decoded token does not contain correct nonce");
-            }
-        } else {
+        if (!decodedToken.contains("request_nonce")) {
```

Comment:
Should also verify that it contains the <nonce?>

Reply:
yes, now it is only looking for nonce